src/lib/alpha.js

```diff
@@ -9,6 +9,7 @@ export const alpha = {
   'es-ES': /^[A-ZÁÉÍÑÓÚÜ]+$/i,
   'fa-IR': /^[ابپتثجچحخدذرزژسشصضطظعغفقکگلمنوهی]+$/i,
   'fr-FR': /^[A-ZÀÂÆÇÉÈÊËÏÎÔŒÙÛÜŸ]+$/i,
+  'fr-CA': /^[A-ZÀÂÆÇÉÈÊËÏÎÔŒÙÛÜŸ]+$/i,
```

Comment:
Since it's the same regex for `fr-FR` and  `fr-CA`  may be we should just assign the value like it was done for `pt-BR` and all arabic locales

Reply:
Of course.